projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss

```diff
@@ -12,14 +12,6 @@
             .mat-cell, a {
                 color: map.value-from($theme, 'row.selected-color');
             }
-    
```

Comment:
If project is using tables only and not forms, how will checkboxes in table look like?

Reply:
inital i guess. then we would need to make custom styles in the project. other, not-good-at-all option is to duplicate styles for checkbox here as well. unless you have some other solution

Reply:
For time being, we need to duplicate styles to ensure backwards compatibility.
For future, we need to gather all duplication and brainstorm what to do with it.